tests/test_objective_funs.py

```diff
@@ -1534,7 +1534,7 @@ def test_boundary_error_print(capsys):
     obj = VacuumBoundaryError(eq, coilset, field_grid=coil_grid)
     obj.build()
 
-    f = np.abs(obj.compute_unscaled(*obj.xs(eq)))
+    f = np.abs(obj.compute_unscaled(*obj.xs(eq, coilset)))
```

Comment:
was the fact that the old way worked before somewhat unintentional?

Reply:
yeah seems like it was an uncaught bug.